MongoDB/containers/0.1.0/mongodb-config/scaling.sh

```diff
@@ -16,6 +19,6 @@ function scaleup {
 }
 
 # Script starts here
-if [ $($DIG A $MONGO_SERVICE_NAME +short | wc -l) -gt 3 ]; then
+if [ $($GIDDYUP ip stringify --delimiter " " | wc -w) -gt 3 ]; then
```

Comment:
I think you are just getting the set scale? `giddyup service scale` should do it I think.